src/Admin/ElectedRepresentative/ElectedRepresentativeAdmin.php

```diff
@@ -426,6 +429,84 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
                     return true;
                 },
             ])
+            ->add('mandates.zone', CallbackFilter::class, [
+                'label' => 'Périmètres géographiques',
+                'show_filter' => true,
+                'field_type' => ModelAutocompleteType::class,
```

Comment:
maybe we could use `ModelAutocompleteFilter` class instead?

Reply:
It's not possible, the class should implement `FormTypeInterface`, that is not the case of `ModelAutocompleteFilter`.